calculator.test.js

```diff
@@ -0,0 +1,105 @@
+const calc = require("./calculator");
+
+describe("Calculator", () => {
+  it("should ignore the numbers greater than 1000", () => {
+    expect(calc(1011, "*", 3)).toBe(3);
+  });
+  it("should throw an error for an invalid operator", () => {
```

Comment:
let's try to organize the tests using more describes blocks
for example add a new describe under the main describe to include validation tests under it 

```suggestion
describe("Calculator", () => {
....
    describe("Inputs Validations", () => {


```

Reply:
Done